compiler/dyno/include/chpl/uast/Reduce.h

```diff
@@ -64,19 +64,17 @@ namespace uast {
 */
 class Reduce final : public Call {
  private:
-
+  static const int opChildNum_ = 0;
+  static const int iterandExprChildNum_ = 1;
 
   Reduce(AstList children)
-      : Call(asttags::Reduce, std::move(children),
-      /*hasCalledExpression*/ false) {
+      : Call(asttags::Reduce, std::move(children), false) {
```

Comment:
Why did we lose the comment indicating what the `false` applies to?

Reply:
Whoops, I don't even remember why I changed that, I think I removed it and then put it back in lol. Should have caught that when reading the diffs, my bad.